docs/installation.md

```diff
@@ -0,0 +1,98 @@
+> ⚠️ This project is very alpha stage. Expect bugs and incomplete documentation.
+
+The following instructions are intended for contributors who want to try Metals
+on a real-world project.
+
+# Global setup
+
+These steps are required once per machine.
+
+## Step 1 - sbt plugin
+
+The server needs to access some metadata about the build configuration. This
+data are produced by an sbt plugin. This plugin is currently not published, so
+you will need to copy paste it on your machine.
+
+Here's the source of the plugin:
+https://github.com/scalameta/metals/blob/master/project/MetalsPlugin.scala
+
+Copy the source to either (depending on your sbt version):
+
+* (sbt 0.13) `~/.sbt/0.13/plugins/MetalsPlugin.scala`
+* (sbt 1.0) `~/.sbt/1.0/plugins/MetalsPlugin.scala`
+
+## Step 2 - build the VSCode extension
+
+The VSCode extension is not yet published on the Marketplace, so you'll need to
+build it locally.
+
+* Make sure you have installed `node`, `npm` and VS Code.
+* Install the
+  ["Scala Syntax"](https://marketplace.visualstudio.com/items?itemName=daltonjorge.scala)
+  plugin in VS Code.
+* `cd vscode-extension`
+* `npm install`
+* `npm run build`
+* `code --install-extension vscode-metals-0.0.1.vsix`
```

Comment:
This will conflict with #218 (`vscode-metals-0.0.1` -> `metals-0.1.0`)

Reply:
Done.